dotCMS/src/main/java/com/dotmarketing/business/VersionableAPIImpl.java

```diff
@@ -690,10 +707,11 @@ public boolean hasLiveVersion(final Versionable versionable) throws DotDataExcep
 
 		if(versionable instanceof Contentlet) {
 
-			final ContentletVersionInfo contentletVersionInfo = this.getContentletVersionInfo
+			final Optional<ContentletVersionInfo> contentletVersionInfo = this.getContentletVersionInfo
                     (versionable.getVersionId(), ((Contentlet) versionable).getLanguageId());
 
-			return (contentletVersionInfo != null && UtilMethods.isSet(contentletVersionInfo.getLiveInode()));
+			return (contentletVersionInfo.isPresent()
```

Comment:
Shouldn't we check for `contentletVersionInfo.get().getLiveInode()` to be set?

Reply:
We are! but it is on the next line